src/interactions/moderation/anti-raid-nuke.ts

```diff
@@ -2,48 +2,164 @@ import { ApplicationCommandOptionType } from 'discord-api-types/v10';
 
 export const AntiRaidNukeCommand = {
 	name: 'anti-raid-nuke',
-	description: 'Bans all members that have joined recently, with new accounts',
+	description: 'Handle raids and nukes',
 	options: [
 		{
-			name: 'join',
-			description: 'How old should a member be for the cybernuke to ignore them (server join date)?',
-			type: ApplicationCommandOptionType.String,
-			required: true,
-		},
-		{
-			name: 'age',
-			description: "How old should a member's account be for the cybernuke to ignore them (account age)?",
-			type: ApplicationCommandOptionType.String,
-			required: true,
-		},
-		{
-			name: 'pattern',
-			description: "A pattern the member's username should match (case insensitive, full match)",
-			type: ApplicationCommandOptionType.String,
-		},
-		{
-			name: 'avatar',
-			description: 'The avatar hash the user or member should have (exact match)',
-			type: ApplicationCommandOptionType.String,
+			name: 'manual',
+			description: 'Manually filters and bans members based on various criterias',
+			type: ApplicationCommandOptionType.Subcommand,
+			options: [
+				{
+					name: 'pattern',
+					description: 'The pattern to match against the username',
+					type: ApplicationCommandOptionType.String,
+				},
+				{
+					name: 'confusables',
+					description: 'Whether to use confusables',
+					type: ApplicationCommandOptionType.Integer,
+					choices: [
+						{ name: 'Off', value: 0 },
+						{ name: 'Only Pattern (Pattern will be resistant to confusables, Default)', value: 1 },
+						{ name: 'Only Filter Members (Filter only members with confusables)', value: 2 },
+						{ name: 'Pattern & Filter Members', value: 3 },
```

Comment:
i'm not too sure what you're trying to convey here
what does "use confusables" mean? remove confusables, or use as-is, with confusables, if "off"? or does "off" turn the confusables off?
what does "only pattern" and "only filter" have to do with confusables?

Reply:
Only Pattern: Make the pattern resistant to confusables
Only Member: Filter only members that have confusables in their username

Reply:
Clarified, but this option can be changed to a simple boolean (Losing the option to filter out members without confusables)

Reply:
I will mark this as resolved, if you want the change to bool just mention me on discord or post an review/comment here